demisto_sdk/commands/validate/validators/PB_validators/PB118_is_input_key_not_in_tasks.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.common.hook_validations.playbook import PlaybookValidator
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+from demisto_sdk.commands.validate.validators.tools import (
+    collect_all_inputs_from_inputs_section,
+    collect_all_inputs_in_use,
+)
+
+ContentTypes = Playbook
+
+
+class IsInputKeyNotInTasksValidator(BaseValidator[ContentTypes]):
+    error_code = "PB118"
+    description = "Validate that all inputs described in the playbooks input section are used in tasks."
+    error_message = "The playbook '{playbook_name}' contains the following inputs that are not used in any of its tasks: {inputs_not_in_use}"
+    fix_message = ""
+    related_field = ""
```

Comment:
```suggestion
    related_field = "input"
```

Reply:
Good to know, did not know how to use it.Fixed.